pkgs/servers/adguardhome/update.sh

```diff
@@ -8,15 +8,21 @@ set -euo pipefail
 dirname="$(dirname "$0")"
 bins="$dirname/bins.nix"
 
-latest_release=$(curl --silent https://api.github.com/repos/AdguardTeam/AdGuardHome/releases/latest)
+latest_release=$(curl --silent "https://api.github.com/repos/AdguardTeam/AdGuardHome/releases/latest")
```

Comment:
FYI this is a noop change

Reply:
True, I've changed that out of habit, as I've made some mistakes with curl and non-quoted URLs. 

Should I revert it? I wouldn't mind to do that.